core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -97,10 +97,9 @@
    */
   private static boolean restore(PageStore pageStore, PageStoreOptions options, MetaStore metaStore,
       CacheEvictor evictor) {
-    LOG.info("Restore PageStore with {}", options);
```

Comment:
Is this line too noisy? I think its helpful to see it

Reply:
reverted with updated message